infra/src/cli/cli.ts

```diff
@@ -109,7 +109,14 @@ const cli = yargs(process.argv.slice(2))
           type: 'boolean',
           default: false,
         })
-        .demandCommand(1, 'You must pass at least one service to run!')
+        .check((argv) => {
+          const svc = argv.services
+          if (svc.length < 1) {
+            throw new Error('You must pass at least one service to run!')
+          } else {
+            return true
+          }
+        })
```

Comment:
Maybe add a comment explaining that `yargs` doesn't support this out of the box :thinking:

Reply:
Comment added :)